### ISSUE TITLE
file.proto File message FileOrId field

### ISSUE THREAD

**Issue by omrishtam** [meateam/file-service#38]
The FileOrId File message field is defined as
`
oneof fileOrId {
    string parent = 8;
    File parentObject = 9;
}
` in [file.proto](https://github.com/meateam/file-service/blob/caedc5a60e3854889c58cd08edd6b9b918a1d415/protos/file.proto#L141).
And there's a defined message for that in [FileOrId message](https://github.com/meateam/file-service/blob/caedc5a60e3854889c58cd08edd6b9b918a1d415/protos/file.proto#L150) which is not used.
It should be replacing the current definition of the field as `FileOrId parent=8;`.
The resulting File message should be:
```
message File {
    string id = 1;
    string key = 2;
    string fullName = 3;
    string type = 4;
    string description = 5;
    string ownerID = 6;
    int64 size = 7;
    FileOrID parent = 8;
    string bucket = 9;
    string createdAt = 10;
    string updatedAt = 11;
}
```

The `file.proto` should be compiled to go using `protoc -I protos/ protos/file.proto --go_out=plugins=grpc:./protos`.

*Please add this command to the README.md so others would know how to do it.*
Fix this bug in a separate branch called `hotfix/file-proto`.

**Comment by omrishtam** [comment]
Closed in #40.

**Comment by Shahar-Y** [comment]
needs to be undone
